media/css/firefox/new/all-test-variations-pt2.less

```diff
@@ -0,0 +1,250 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+// Common
+#test-primary-ctas {
+    ul {
+        list-style: none;
+    }
+
+    li {
+        display: inline-block;
+        margin: 0;
+    }
+}
+
+#test-mobile-downloads {
```

Comment:
It looks like these mobile links need to be block-level and have a set height/width to show up on IE < 9